docs/02-get-started/04-trigger-workload-with-event.md

```diff
@@ -118,16 +116,9 @@ cat <<EOF | kubectl apply -f -
      name: lastorder-sub
      namespace: default
    spec:
-     filter:
-       filters:
-       - eventSource:
-           property: source
-           type: exact
-           value: ""
-         eventType:
-           property: type
-           type: exact
-           value: sap.kyma.custom.myapp.order.received.v1
+     source: myapp
```

Comment:
don't forget to change the `apiVersion` to `eventing.kyma-project.io/v1alpha2`.

Reply:
Updated.